### ISSUE TITLE
Dropping CLViewController dependency

### ISSUE THREAD

**Issue by bejo** [appunite/CLCascade#33]
Hi Emil!
First of all, congratulations on Cascade! Your Cascade control is awesome!

Recently, I forked it to allow the use of plain UIViewControllers (not only CLViewController subclasses). Along the way it turned out that dropping support for iOS4 would make this task easier. In the end, I refactored a lot classes and simplified few interfaces - I had really good reasons for doing and that I'd be happy to discuss it with you (ping me on Twitter: https://twitter.com/_bejo).

Long story short, it would be awesome if you could merge my changes into you repository. I haven't send the pull request because I'm not quite done yet, but you can have a sneak peek view here: https://github.com/futuresimple/CLCascade

Let me know what you think about it as your decision will affect my further work on the control. Cheers! :)

**Comment by emilwojtaszek** [comment]
sounds great! waiting for your pull requests!

**Comment by bejo** [comment]
Done: https://github.com/appunite/CLCascade/pull/35
